ndk/src/media/mod.rs

```diff
@@ -3,20 +3,13 @@
 //! See also [the NDK docs](https://developer.android.com/ndk/reference/group/media)
 #![cfg(feature = "media")]
 
-mod error;
```

Comment:
That might be a breaking change; but we have more breaking PRs queued up for `ndk` anyway.

Reply:
Never mind, it's not `pub` so this module was never visible.